### DISCUSSION TITLE
Javascript version (closed)

### DISCUSSION THREAD

**Discussion by wgmayer0** [junegunn/fzf#2454] [General]
FZF is one of the greatest pieces of software ever written. The algorithm is so good. Is there anyway to use it with Javascript? I am looking to quickly select data from an array of possible candidates to store as a string in a database.

**Comment by wgmayer0** [comment]
I see it is written in Go so I know what to do: I would just copy the exact code in Go and then change the syntax to Javascript. That would answer it!

**Reply by aehlke** [reply]
can you share?

**Comment by wgmayer0** [comment]
I was just trolling to get junegunn to respond, lol. Looks like I failed
haha.

I did find a solution though for anyone wondering. There is an
implementation done in Javascript and Typescript:
https://github.com/ajitid/fzf-for-js

You want to use https://www.skypack.dev/ with it to get it to work in your
browser.

**Reply by aehlke** [reply]
ah heh I have translated libs 1:1 like that before to another language/platform out of desperation so your joke is not so absurd to me lol. personally I will be using minisearch for js

**Comment by wgmayer0** [comment]
🤣🤣🤣
